README.md

```diff
@@ -25,6 +25,23 @@ See the [Electron docs for using native modules][electron-native-modules] to ens
 [electron-rebuild]: https://github.com/electron/electron-rebuild
 [electron-native-modules]: https://www.electronjs.org/docs/tutorial/using-native-node-modules
 
+### Known problems using with electron:
```

Comment:
I think this heading might be a little misleading and unnecessary. There aren't problems using this library with Electron (it works great!). If you encounter this error, it means _your own project_ isn't set up correctly

Reply:
Thank you, I agree with you and going to change that!